src/app/products/manage-tax-groups/create-tax-group/create-tax-group.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="container">
+<mat-card>
+<form [formGroup]="TaxGroupForm">
+  <mat-card-title>
+    <h3>Create Tax Group</h3> <hr>
+  </mat-card-title>
+  <mat-card-content>
+    <span class="namefield">Name*
+      <mat-form-field appearance="outline">
+        <input matInput required formControlName='name'>
+        <mat-error *ngIf="TaxGroupForm.controls.name.hasError('required')">
+          Name is <strong>required</strong>
+        </mat-error>
+      </mat-form-field>
+    </span>
+
+    <h1>Tax Components</h1> <br>
+    <div fxLayout="row wrap" fxLayoutGap="30%" fxLayout.lt-md="column">
+       <h2>Tax Component</h2>
+        <h2>Start Date</h2>
```

Comment:
use <mat-label> for Start Date and place it in the <mat-form-field>

Reply:
> use for Start Date and place it in the

@Rishabh106  couldn't get it.can you please elaborate?

Reply:
Edited comment, please check again.

Reply:
> use mat-label for Start Date and place it in the mat-form-field

But this will make the word "start date" appear as many times as user presses the add button.

Reply:
Sorry, my bad, don't use an h2 tag there, use class as mat-body-strong in the div of title.

Reply:
> Sorry, my bad, don't use an h2 tag there, use class as mat-body-strong in the div of title.

@Rishabh106  Do you mean creating a div with class="mat-body-strong" in place of h2 tag?

Reply:
Yes